samples/tfm/tfm_secure_peripheral/boards/nrf54lm20dk_nrf54lm20a_cpuapp_ns.conf

```
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# P1: 5 (SPIM_MOSI), 6 (SPIM_SCK) 15 (button0)
+CONFIG_NRF_GPIO1_PIN_MASK_SECURE=0x00002060
```

This should probably be
```suggestion
CONFIG_NRF_GPIO1_PIN_MASK_SECURE=0x00008060
```
and in `nrf54lv10dk_nrf54lv10a_cpuapp_ns.conf` as well (assuming the comment above is correct).

---

So update the comments?

---

Yes, that is indeed the plan.